schemachange/module.mk

```diff
@@ -17,7 +17,8 @@ schemachange/sc_records.o	\
 schemachange/sc_schema.o	\
 schemachange/sc_stripes.o	\
 schemachange/sc_struct.o	\
-schemachange/sc_util.o
+schemachange/sc_util.o      \
+db/comdb2uuid.o
```

Comment:
it uses that to check uuid in resume schema change.. and libschemachange.a needs it